app/assets/stylesheets/admin/loans/timeline/_table.scss

```diff
@@ -129,6 +133,10 @@
     overflow: auto;
     min-height: 30em;
   }
+
+  td.project-group:hover {
+    background: $background-gray;
```

Comment:
`$background-gray` is very light and `$gray3` is very dark. Let's use the same hover color for project group and project step for consistency.

For now, let's use the color that is the hover color for items in other tables.

```
#cfcfcf  / rgb (207,207,207)
```

Reply:
This color will need to be added as a variable in the colors stylesheet. You don't need to add it now, but it needs to be added before we merge in this ticket.